src/api/app/views/webui/users/_breadcrumb_items.html.haml

```diff
@@ -1,18 +1,17 @@
-= render partial: 'webui/main/breadcrumb_items'
 - case action_name
   - when 'index'
+    = render partial: 'webui/configuration/breadcrumb_items'
     %li.breadcrumb-item.active{ 'aria-current' => 'page' }
       Manage Users
   - when 'new'
+    = render partial: (User.session ? 'webui/configuration/breadcrumb_items' : 'webui/main/breadcrumb_items')
```

Comment:
Is the ternary operator needed? I believe it should always be `'webui/configuration/breadcrumb_items'`.